### PR TITLE
NCP-2875 and the advanced property search

### PR DESCRIPTION
**Pull request by marcfishelquantil**
Burton, please comment on the text and ignore missing or old GUI screenshots. I'll work with Eleazar on the screenshots.

### CODE REVIEW

File: en/docs/portal/edge-configurations/managing-properties.md [hunk 1]
```diff
@@ -32,7 +32,26 @@ CDN Pro provides robust filtering capabilities that allow you to find properties
...
-If you have many properties, use <b>hasConfig=<i><'configuration'></i></b> to find properties that match a specific configuration. In an advanced search string, <b><i><'configuration'></b></i> is a keyword supported by the CDN Pro API. For more information, refer to the [CDN Pro API Specification](</apidocs#operation/createPropertyVersion>).<br>
+CDN Pro provides an advanced search filter that allows you to find properties that match a specific configuration. This feature is particularly useful if you have many properties. 
+
+You can access the advanced search using either the Advanced Filter icon next to the search box or the <b>hasConfig=<i><'configuration'></b></i> search string syntax.
+
+#### Using the Advanced Filter Icon
+The Advanced Filter icon next to the search box allows you to search specific configurations by entering keywords and conditions in an Advanced Filter dialog box.
+1. Click the Advanced Filter icon.
+   
+   <p align=center><img src="/docs/resources/images/edge-configurations/advanced-filter-icon.png" alt="advfilter icon" width="900"></p>
```
Comment (burtonquantil) [inline en/docs/portal/edge-configurations/managing-properties.md:43]: I don't see the image file in your pull request.

File: en/docs/portal/edge-configurations/managing-properties.md [hunk 2]
```diff
@@ -32,7 +32,26 @@ CDN Pro provides robust filtering capabilities that allow you to find properties
...
+2. When the dialog box appears, complete the following fields.
+
+| **Fields/Buttons**   | **Description**                                                                           |
+| :----------: | ----------------------------------------------------------------------------------------- |
+| Keywords     | To search by keywords, enter one or more keywords associated with the property you want to find. A keyword can be a property ID, name, or description. To enter multiple keywords, separate each one with a carriage return. For more information, refer to the [CDN Pro API Specification](</apidocs#operation/createPropertyVersion>).<br>                                   |
+| Add New +    | To search based on conditions, click **Add New +**. Select a field name and operand from the first two lists. If a value is required, enter it in the rightmost field. Click **Add New +**. Repeat these steps for each additional filter condition you want to add.               |
```
Comment (burtonquantil) [inline en/docs/portal/edge-configurations/managing-properties.md:49]: Are you sure "operand" is the right word? That is something to be operated on.
We shouldn't use "operator"?

File: en/docs/portal/traffic-management/creating-edge-hostname.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ An edge hostname in a CNAME record on your DNS system allows you to route end us
...
+| Server Group      | If <strong>Action Type</strong> is set to <strong>Deliver</strong>, select one or more server groups to deliver the content for this client zone. CDN Pro servers are organized into 5 groups based on traffic cost from low to high: <strong>Standard</strong>, <strong>Premium</strong>, <strong>Premium+</strong>, <strong>Ultra</strong>, and <strong>nearChina</strong>. When you select one group, all the lower groups will be automatically included. <br><strong>Note:</strong> <strong>nearChina</strong>, <strong>ChinaBase</strong>, and <strong>ChinaUltra</strong> are special server groups that can be enabled for you if you want optimal performance serving Chinese visitors, but your domains lack ICP Beian. To enable these server groups, contact the CDNetworks support team.</br><br>For more information, see [About Server Groups](#about-server-groups) below. To see the location of servers in each group, go to: https://www.cdnetworks.com/cdnpro/.|
```
Comment (burtonquantil) [inline en/docs/portal/traffic-management/creating-edge-hostname.md:31]: There are more than 5 server groups now.

ChinaBase and ChinaUltra are for those customers WITH ICP Beian license for their hostnames and want the content served from servers in China.
Only NearChina is for those without ICP Beian and want better performance.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Aside from updating images, you will also need to remove "Need this hostname to be resolved to edge servers in mainland China" in traffic-management/creating-edge-hostname

I see no Advanced Settings in edge hostnames.
Remove from https://docs.cdnetworks.com/en/cdn/docs/portal/traffic-management/creating-edge-hostname and https://docs.cdnetworks.com/en/cdn/docs/portal/traffic-management/editing-edge-hostname ?

File: en/docs/portal/edge-configurations/managing-properties.md (hunk 1 above)
Comment (marcfishelquantil) [reply en/docs/portal/edge-configurations/managing-properties.md:43]: I made the text changes you noted above. The image you flagged is one I will need Eleazar to capture for me.

File: en/docs/portal/edge-configurations/managing-properties.md (hunk 2 above)
Comment (marcfishelquantil) [reply en/docs/portal/edge-configurations/managing-properties.md:49]: Changed to "operator."

File: en/docs/portal/traffic-management/creating-edge-hostname.md (hunk 3 above)
Comment (marcfishelquantil) [reply en/docs/portal/traffic-management/creating-edge-hostname.md:31]: I made these changes, thank you.

**Review by burtonquantil — APPROVED** [review]
OK to merge.

**Pull request merged**